src/main/java/io/jenkins/plugins/synopsys/security/scan/PluginParametersHandler.java

```diff
@@ -42,7 +41,7 @@ public int initializeScanner(Map<String, Object> scanParameters) throws PluginEx
 
         logMessagesForParameters(scanParameters, scanParametersService.getSynopsysSecurityProducts(scanParameters));
 
-        int exitCode = -1;
+        int exitCode = 31; // Scan parameter validation failure error code
```

Comment:
Souldn't we use PARAMETER_VALIDATION_FAILED  from ErrorCode class instead of 31?

Reply:
Yes we should